File: hw/nacato/u3/hw23AbstractFactory/hw23AbstractFactory.java/src/main/java/ec/edu/espe/AbstractFactory/model/LinuxFactory.java

package ec.edu.espe.AbstractFactory.model;

/**
 *
 * @author Angie Ñacato, Error 404, @ESPE
 */
public class LinuxFactory extends GUIFactory {
    @Override
    public Button createButton() {
        return new LinuxButton();
    }
    @Override
    public Menu createMenu() {
        return new LinuxMenu();
    }
}

File: hw/chavez/u3/hw24AbstractFactory/JavaScript/model/LinuxFactory.js
const GUIFactory = require("./GUIFactory");
const LinuxButton = require("./LinuxButton");
const LinuxMenu = require("./LinuxMenu");

class LinuxFactory extends GUIFactory {

    createButton() {
        return new LinuxButton();
    }

    createMenu() {
        return new LinuxMenu();
    }
}

module.exports = LinuxFactory;